wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WContainer.java

```diff
@@ -92,8 +67,6 @@ public void setTemplate(final String templateUrl) {
 	}
 
 	/**
-	 * {@inheritDoc}
```

Comment:
Please retain this inheritDoc tag as the inheritance needs to be explicit due to the other comments in the block.